vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/openid/PresentationFactory.kt

```diff
@@ -119,9 +124,14 @@ internal class PresentationFactory(
             docType = docType,
             namespaces = deviceNameSpaceBytes
         )
-        Napier.d("Device authentication is ${deviceAuthentication.serialize().encodeToString(Base16())}")
+        // TODO Change type to CoseSigned<DeviceAuthentication>
```

Comment:
why todo?

Reply:
Rather large change ... so it would keep it as it is for now